rfd/0086/compat.md

```diff
@@ -0,0 +1,7 @@
+## Backwards compatibility and support
+
+The changes in this RFD are not backwards compatible with ContainerPilot 2.x configuration. For end users using Consul as their discovery backend, each container image can be safely updated independently; there is no communication between ContainerPilot instances except through the Consul API.
```

Comment:
Yeah, good point. Here are my initial thoughts there:

- We're not breaking (1) but any application that relies on ordering of registration will probably be broken by (3) and (4)
- We're leaving (2) alone.
- We're proposing a complete break of (3). The config language is different, and we don't have `coprocess` or `task` or `preStart`.
- The elimination of `coprocess`, `task`, and `preStart` in lieu of the explicit dependency management will probably break most existing implementations that aren't totally trivial.

Also missing from this section is what we'll do from a Joyent Support team perspective with ContainerPilot 2.x.